src/splitInteger.test.js

```diff
@@ -2,20 +2,36 @@
 
 const splitInteger = require('./splitInteger');
 
-test(`should split a number into equal parts
-  if a value is divisible by a numberOfParts`, () => {
-
-});
-
-test(`should return a part equals to a value
-  when splitting into 1 part`, () => {
-
-});
-
-test('should sort parts ascending if they are not equal', () => {
-
-});
-
-test('should add zeros if value < numberOfParts', () => {
-
+describe('#splitInteger', () => {
+  test(`should split a number into equal parts
```

Comment:
make your life easier:
test(`should split a number into equal parts
  if a value is divisible by a numberOfParts`, () => {
  expect(splitInteger(8, 4))
    .toEqual([2, 2, 2, 2]);
});